VTEX - Orders API.json

```diff
@@ -2218,9 +2209,7 @@
         },
         "/api/oms/pvt/orders/{orderId}/start-handling": {
             "post": {
-                "tags": [
-                    "Orders"
-                ],
+                "tags": ["Orders"],
                 "summary": "Start handling order",
                 "description": "Change the status of an order to indicate that is is in `handling`.\n\r\n\r> Expect a `status 204` response with no content in case of a successful request. The store must validate this response to retry the call if the responses differs from the 204 code, making this flow the store's responsibility. This endpoint can also respond with `status 500`. \n\r\n\r> The `Change order workflow status` resource is needed to use this API request. This is included in `OMS - Full access` and `IntegrationProfile - Fulfillment Oms`, among other default roles available in the Admin. Learn more about the [License manager roles and resources](https://help.vtex.com/en/tutorial/roles--7HKK5Uau2H6wxE1rH5oRbc#).",
```

Comment:
```suggestion
                "description": "Changes the status of an order to indicate that it is in `handling`.\n\r\n\r> Expect a `status 204` response with no content in case of a successful request. The store must validate this response to retry the call if the response differs from the `204` code, making this flow the store's responsibility. This endpoint can also respond with `status 500`. \n\r\n\r> The `Change order workflow status` resource is needed to use this API request. This is included in `OMS - Full access` and `IntegrationProfile - Fulfillment Oms`, among other default roles available in the Admin. Learn more about the [License manager roles and resources](https://help.vtex.com/en/tutorial/roles--7HKK5Uau2H6wxE1rH5oRbc#).",
```